packages/perennial/contracts/periphery/CoordinatorDelegatable.sol

```diff
@@ -18,88 +20,229 @@ contract CoordinatorDelegatable is UOwnable {
+    );
+    function makerFeeLimits() public view returns (UFixed18ParamLimits memory) { return _makerFeeLimits.read(); }
+
+    /// @dev The taker fee value limits
+    UFixed18ParamLimitsStorage private constant _takerFeeLimits = UFixed18ParamLimitsStorage.wrap(
+        keccak256("equilibria.perennial.CoordinatorDelegatable.takerFeeLimits")
+    );
+    function takerFeeLimits() public view returns (UFixed18ParamLimits memory) { return _takerFeeLimits.read(); }
+
+    /// @dev The maker limit value limits
+    UFixed18ParamLimitsStorage private constant _makerLimitLimits = UFixed18ParamLimitsStorage.wrap(
+        keccak256("equilibria.perennial.CoordinatorDelegatable.makerLimitLimits")
+    );
+    function makerLimitLimits() public view returns (UFixed18ParamLimits memory) { return _makerLimitLimits.read(); }
+
+    /// @dev The utilization buffer value limits
+    UFixed18ParamLimitsStorage private constant _utilizationBufferLimits = UFixed18ParamLimitsStorage.wrap(
+        keccak256("equilibria.perennial.CoordinatorDelegatable.utilizationBufferLimits")
+    );
+    function utilizationBufferLimits() public view returns (UFixed18ParamLimits memory) { return _utilizationBufferLimits.read(); }
+
+    /// @dev The utilization curve limits
+    UtilizationCurveLimitsStorage private constant _utilizationCurveLimits = UtilizationCurveLimitsStorage.wrap(
+        keccak256("equilibria.perennial.CoordinatorDelegatable.utilizationCurveLimits")
+    );
+    function utilizationCurveLimits() public view returns (UtilizationCurveLimits memory) { return _utilizationCurveLimits.read(); }
+
+    /**
+     * @notice Constructors a new CoordinatorDelegatable
+     * @param _product The product this coordinator is for
+     */
+    constructor(IProduct _product) {
+        product = _product;
+    }
+
     /**
      * @notice Initializes the coordinator owner contract
+     * @param paramAdmin_ The address of the param admin
+     * @param maintenanceLimits_ The maintenance value limits
+     * @param makerFeeLimits_ The maker fee value limits
+     * @param takerFeeLimits_ The taker fee value limits
+     * @param makerLimitLimits_ The maker limit value limits
+     * @param utilizationBufferLimits_ The utilization buffer value limits
+     * @param utilizationCurveLimits_ The utilization curve limits
      * @dev Sets the deployer as the default admin and param admin
      */
-    function initialize() public initializer(1) {
+    function initialize(
+        address paramAdmin_,
+        UFixed18ParamLimits memory maintenanceLimits_,
+        UFixed18ParamLimits memory makerFeeLimits_,
+        UFixed18ParamLimits memory takerFeeLimits_,
+        UFixed18ParamLimits memory makerLimitLimits_,
+        UFixed18ParamLimits memory utilizationBufferLimits_,
+        UtilizationCurveLimits memory utilizationCurveLimits_
+    ) external initializer(1) {
         __UOwnable__initialize();
+        updateParamAdmin(paramAdmin_);
```

Comment:
I kind of like the pattern of not setting any parameters in initialize -- seems a bit cleaner cuz we can always call the functions directly to set stuff up separately